README_OPENTELEMETRY.rst

```diff
@@ -0,0 +1,45 @@
+Metrics with OpenTelemetry
+==================================
+This library uses `OpenTelemetry <https://opentelemetry.io/>`_ to record metric information on API requests.
+For information on the benefits and utility of tracing, see the `Cloud Monitoring docs <https://cloud.google.com/monitoring/docs>`_.
```

Comment:
nit: this PR adds metrics but not tracing yet; is "tracing" the right word here?